composer.json

```
@@ -0,0 +1,24 @@
+{
+    "name": "mutualaidnyc/wp-theme",
+    "description": "WordPress theme for Mutual Aid NYC.",
+    "type": "wordpress-theme",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Echo G",
+            "email": "ChaosExAnima@users.noreply.github.com"
+        }
+    ],
+    "require": {},
+    "require-dev": {
+        "squizlabs/php_codesniffer": "^3.5",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.6.2",
+        "wp-coding-standards/wpcs": "^2.2",
+        "sirbrillig/phpcs-changed": "^2.4",
+        "sirbrillig/phpcs-variable-analysis": "^2.8"
+	},
+	"scripts": {
+		"php:lint": "vendor/bin/phpcs -p -s functions.php",
+		"php:changed": "vendor/sirbrillig/phpcs-changed/bin/phpcs-changed --git"
```

Should this line have a file or glob that follows it? e.g.

```suggestion
		"php:changed": "vendor/sirbrillig/phpcs-changed/bin/phpcs-changed --git *.php"
```

---

Not sure, honestly? Copied that from the Jetpack repo but I'm unfamiliar with that script.

---

It appears to lint files staged to git, if you give it some paths at the end.

In any case, I'm good with this to get some basic linting going!